hotel/site_sections/hotel_assignments.py

```diff
@@ -138,6 +138,23 @@ def ordered(self, out, session):
             for a in group:
                 writerow(a, a.hotel_requests)
 
```

Comment:
I think this is pretty straightforward (i.e. it looks basically the same as all of our other ``@csv_file`` pages) but let me know if there are any questions.